src/tests/CMakeLists.txt

```
@@ -38,4 +38,8 @@ foreach(PROG
   # run tests for tools (for coverage) as well
   set_tests_properties(unit_${PROG} PROPERTIES LABELS "tools;votca")
 
+  if(VALGRIND_EXECUTABLE)
```

```suggestion
  if(VALGRIND_FOUND)
```
`find_package_handle` defines that.